code/modules/mob/living/carbon/xenomorph/death.dm

```diff
@@ -45,6 +45,8 @@
 	if (isAI)
 		gib()
 
+	to_chat(src,"<b><span class='deadsay'><p style='font-size:1.5em'><big>We have perished.</big><br><small>But it is not the end of us yet... wait until a newborn can rise in this world...</small></p></span></b>")
```

Comment:
My recommendation about this was to add a `special_death_message` into the caste datum for each xeno. But this is also acceptable.